R/utils.R

```diff
@@ -579,6 +579,16 @@ get_plot_coords <- function(plot) {
   diff(ggplot2::layer_scales(plot)$x$get_limits())
 }
 
+#' Changes dashes in character vectors for correct plotting.
```

Comment:
Pls add another sentence: en-dashes will be subbed by "\uad", so they are displayed correctly in the plotted PDF. 

Background: The plots for the Bildungstrend will be saved as PDF. However, the graphics device will use em-dashes by default (which are longer then en-dashes). Therefore, we have to sub them manually with this code, so they are displayed correctly in the PDF.